src/main/java/io/lbry/browser/reactmodules/BackgroundMediaModule.java

```diff
@@ -0,0 +1,91 @@
+package io.lbry.browser.reactmodules;
+
+import android.app.Activity;
+import android.app.Notification;
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.os.Build;
+import android.support.v4.app.NotificationCompat;
+import android.support.v4.app.NotificationManagerCompat;
+import android.support.v4.content.ContextCompat;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+
+import io.lbry.browser.MainActivity;
+import io.lbry.browser.R;
+
+public class BackgroundMediaModule extends ReactContextBaseJavaModule {
+
+    private static final int NOTIFICATION_ID = 900;
+
+    private static final String NOTIFICATION_CHANNEL_ID = "io.lbry.browser.MEDIA_PLAYER_NOTIFICATION_CHANNEL";
+
+    public static final String ACTION_PLAY = "io.lbry.browser.ACTION_MEDIA_PLAY";
+
```

Comment:
Some of these line breaks aren't needed here

Reply:
I usually format this way to improve readability. Also helps when adding Javadocs.